common/windows/delphi/tools/sentrytool/build.sh

```diff
@@ -30,3 +31,4 @@ builder_run_action clean:project        clean_windows_project_files
 builder_run_action configure:project    configure_windows_build_environment
 builder_run_action build:project        delphi_msbuild sentrytool.dproj "//p:Platform=Win32"
 # builder_run_action test:project         do_test
+builder_run_action edit:project         start sentrytoolTests.dproj
```

Comment:
Why does this open `sentrytoolTests.dproj` instead of `sentrytool.dproj`?

Reply:
because I was cross-eyed when I wrote it?

```suggestion
builder_run_action edit:project         start sentrytool.dproj
```